src/Components/Login.tsx

```diff
@@ -1,13 +1,39 @@
 import React, { useContext, useState } from 'react';
 import { UserContext } from '../Contexts/UserContext';
+import { registerUser } from '../network';
 
 export default function Login() {
+	const { setUsername, setHome } = useContext(UserContext);
 	const [formUsername, setFormUsername] = useState<string>('');
 	const [latitude, setLatitude] = useState<string>('');
 	const [longitude, setLongitude] = useState<string>('');
+	const [errorMessage, setErrorMessage] = useState<string>('');
+
+	function isValidLat(input: string): boolean {
+		const value = parseFloat(input);
+		return !isNaN(value) && value >= -90 && value <= 90;
+	}
+
+	function isValidLong(input: string): boolean {
+		const value = parseFloat(input);
+		return !isNaN(value) && value >= -180 && value <= 180;
+	}
```

Comment:
We need some error messages for this! I put in `aaaaaa` and was confused because there was no message :b

Reply:
OK, will work on tomorrow

Reply:
Is this OK now or did you need more?

Reply:
For an MVP sure